### ISSUE TITLE
Sidebar blocks should always have a heading and section

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#5176]
Our accessibility audit recommended that all sidebar blocks have a `<h2>` heading. In addition, each block should be wrapped in a `<section>` instead of a `<div>`.

For example, the language block currently looks like this:

```html
<div class="pkp_block block_language">
	<span class="title">
		Language
	</span>
	<div class="content">...</div>
</div>
```

This should be:

```html
<section class="pkp_block block_language">
	<h2 class="title">
		Language
	</h2>
	<div class="content">...</div>
</section>
```

This will need to be changed for _all of our core blocks_. 

In some cases, a block may only have the `<div class="content">...</div>` with no title section. In such cases, a title must be added with a title that is hidden from sighted users but available to screen readers.

Finally, the same changes should be made to all blocks of content on the article landing page. For example, the list of authors should be placed in a `<section>` and given a `<h2>Authors</h2>` heading, and that heading should be hidden from sighted users but available to screen readers.

**Comment by NateWr** [comment]
When adjusting the blocks on the article landing page, make sure that the primary download files and the supplementary files each have hidden `<h2>` elements which describe them.

Primary download files should say "Files" and the supplementary section should say "Additional Files".

**Comment by thinkbulecount2** [comment]
PRs:
[OJS](https://github.com/pkp/ojs/pull/2677)
OMP

Plugins
[customBlockManager](https://github.com/pkp/customBlockManager/pull/62)
[usageStatus](https://github.com/pkp/usageStats/pull/26)
[makeSubmission](https://github.com/pkp/makeSubmission/pull/32)
[browse](https://github.com/pkp/browse/pull/2)

**Comment by NateWr** [comment]
These look good to go. I just added a couple more comments but they're all good. I'm going to hold off on the merge again until 3.1.2-1 is released.

**Comment by NateWr** [comment]
All merged!

**Comment by asmecher** [comment]
There are cases where `article_summary.tpl` is used without the new `$heading` template variable being set, leading to broken article summaries e.g. in the search results:
![image](https://user-images.githubusercontent.com/200411/80229593-81459180-8605-11ea-9f74-18822a329311.png)
I think the `$heading` variable should have a default in the template, and the variable should be documented in the self-documentation at the head of the file.

**Comment by thinkbulecount2** [comment]
Hi @asmecher and @NateWr is this enough or should I add more information? We are assuming that the `$heading` variable starts at `h2` in most cases.
```
{* Assigning "h2" as the default heading for sub-titles, if none specified *}
{if !$heading}
	{assign var="heading" value="h2"}
{/if}
```

We are doing the same for OMP.

**Comment by thinkbulecount2** [comment]
Also the same logic exists for `templates/frontend/objects/announcement_summary.tpl`

`{if !$heading}
	{assign var="heading" value="h2"}
{/if}`

**Comment by NateWr** [comment]
Yes, that logic looks good. I see that it's documented at the top of `monographList.tpl`, like this:

```
@uses $heading string HTML heading element, default: h2
```

As Alec suggested, we should also document it in `monograph_summary.tpl` and `article_summarty.tpl` (and `issue_toc.tpl` when added to that).

**Comment by thinkbulecount2** [comment]
Made the change here: https://github.com/pkp/ojs/pull/2692/commits/54458a8dc68e352b43bd7d85b8f0661ee61c978c as part of https://github.com/pkp/ojs/pull/2692

To be done in OMP and OPS as well

**Comment by NateWr** [comment]
The remaining parts of this are in #5178 so I'm closing this again in favor of discussing further there.
